engine/runtime/src/main/java/org/enso/interpreter/runtime/type/Types.java

```diff
@@ -157,6 +154,10 @@ public static String getName(Object value) {
       return ConstantsGen.PANIC;
     } else if (TypesGen.isPanicSentinel(value)) {
       return ConstantsGen.PANIC;
+    } else if (TypesGen.isWarning(value)) {
```

Comment:
`TypesGen.isXYZ` checks are problematic and we should plan to get rid of them. That's however task for another PR.